internal/provider/models/processors/base_model.go

```diff
@@ -44,13 +44,16 @@ var baseProcessorSchemaAttributes = SchemaAttributes{
 	},
 }
 
-func ExtendBaseAttributes(target SchemaAttributes) SchemaAttributes {
-	return ExtendSchemaAttributes(baseProcessorSchemaAttributes, target)
+// ExtendBaseAttributes returns a new map with the base attributes and target
+func ExtendBaseAttributes(target map[string]schema.Attribute) map[string]schema.Attribute {
```

Comment:
For consistency, I don't think we should change the name of this function.  "extend" and "merge" are synonyms, and we use `ExtendBaseAttributes` in the `base_model.go` of other resource types.

Reply:
Fixed, this code snug in as i was trying to solve and reason about the tolist() function. Since that has been fixed now -- it was a bug in Terraform v1.1 -- this code has been dropped from the PR.